lib/util/fs/index.ts

```diff
@@ -300,3 +300,10 @@ export async function getLocalFiles(
 
   return fileContentMap;
 }
+
+const validFilePathRegex = new RegExp(
+  /^(?!\/)(\.{1})?([a-z0-9/]+)?(\/\.{3})?$/
```

Comment:
When I look at this regex, it's very hard to decide "is this right? is this safe?"

Reply:
sure, I'll break it down:
```
^                       // sting start
(?!\/)                 // does not have a leading slash '/'
(\.{1})?             // may or may not have a leading dot '.'
([a-z0-9/]+)?   // an arbitrary amount of alphanumeric chars, or slash `/`
(\/\.{3})?         // may or may not finish with triple dot '...'
$                    // string end
```
Hope that's clearer. I can add a description like this to the code if that helps.

Should be  safe, at least I couldn't think of a way to break it.

Reply:
Maybe add that description as a comment in the code.

Also, any chance you could explain to me how `(\.{1})?` is different from `\.?`?

Reply:
it's the same thing :upside_down_face: 
updated the PR with the description. Any chance we can get this merged soon?